outputs.tf

```diff
@@ -1,19 +1,23 @@
 // The 'writer' endpoint for the cluster
 output "cluster_endpoint" {
-  value = "${join("", aws_rds_cluster.default.*.endpoint)}"
+  value = join("", aws_rds_cluster.default.*.endpoint)
 }
 
 // Comma separated list of all DB instance endpoints running in cluster
```

Comment:
This comment says it's a comma separated list which is wrong. If it were true, then I think this module could be made compatible with both 0.11.x and 0.12.x.